### PR TITLE
Animal shelter

### PR DESCRIPTION
**Pull request by rnmessick**
- [x] Top-level README “Table of Contents” is updated
 - [ ] Feature tasks for this challenge are completed
 - [ ] Unit tests written and passing
     - [ ] “Happy Path” - Expected outcome
     - [ ] Expected failure
     - [ ] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [x] Picture of whiteboard

### CODE REVIEW

File: code401challenges/src/main/java/code401challenges/utilities/AnimalShelter.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    public Queue<T> dog;
+    public Queue<T> cat;
+
+    public AnimalShelter() {
+        this.dog = new Queue<>();
+        this.cat = new Queue<>();
+    }
+
+    public void enqueue(T animal) {
+        if(animal.equals(dog)) {
```
Comment (evanslaton) [inline code401challenges/src/main/java/code401challenges/utilities/AnimalShelter.java]: `dog` was assigned a `Queue` so unless the 'thing' being passed into this method is a queue this conditional statement won't work

File: code401challenges/src/main/java/code401challenges/utilities/AnimalShelter.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+            this.dog.enqueue(animal);
+        }
+        else {
+            animal.equals(cat);
```
Comment (evanslaton) [inline code401challenges/src/main/java/code401challenges/utilities/AnimalShelter.java]: Same as previous comment regarding `cat`

File: code401challenges/src/test/java/code401challenges/utilities/AnimalShelterTest.java [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+import static org.junit.Assert.*;
+
+public class AnimalShelterTest {
+
+    @Test
+    public void testEnqueueAnimalShelter() {
+        AnimalShelter testShelter = new AnimalShelter();
+        Queue dog = new Queue<>();
+
+        testShelter.enqueue(dog);
```
Comment (evanslaton) [inline code401challenges/src/test/java/code401challenges/utilities/AnimalShelterTest.java]: Since `dog` is a `queueu` these two lines are enqueue-ing queues into the animal shelter
